Library/Homebrew/os/mac/xcode.rb

```diff
@@ -27,6 +27,11 @@ def latest_version
         end
       end
 
+      def prerelease?
+        # TODO: bump version after Xcode 8.0 is stable.
+        version > "7.3.1"
```

Comment:
We haven't got above a `x.3.1` since OS X 10.7, it felt safe enough, but my initial writing was `>= "8.0"` and then I disagreed with myself.

Plan for once Xcode 8 is stable is simply `version > "8.0"`. We have to update Xcode code every release anyway, it only adds an extra line to remember.